Library/Formula/crystal-lang.rb

```diff
@@ -0,0 +1,61 @@
+class CrystalLang < Formula
+  desc "Efficient, type safe and statically compiled language with Ruby inspired syntax"
```

Comment:
Please shorten this description. “`crystal-lang: <desc>` should fit under 80 chars.

```
==> audit problems
crystal-lang:
 * Description is too long. "name: desc" should be less than 80 characters.
Length is calculated as crystal-lang + desc. (currently 93)
```